crates/sncast/tests/helpers/fixtures.rs

```diff
@@ -79,41 +79,83 @@ pub async fn deploy_keystore_account() {
     let keystore_path = "tests/data/keystore/predeployed_key.json";
     let account_path = "tests/data/keystore/predeployed_account.json";
     let private_key =
-        SigningKey::from_keystore(keystore_path, "123").expect("Could not get the private_key");
+        SigningKey::from_keystore(keystore_path, "123").expect("Failed to get private_key");
 
-    let provider = get_provider(URL).expect("Could not get the provider");
+    let contents = fs::read_to_string(account_path).expect("Failed to read keystore account file");
+    let items: Value = serde_json::from_str(&contents)
+        .unwrap_or_else(|_| panic!("Failed to parse keystore account file at = {account_path}"));
+
+    let deployment_info = items
+        .get("deployment")
+        .expect("Failed to get deployment key");
+    let address = get_from_json_as_str(deployment_info, "address");
+
+    deploy_account_helper(
+        address,
+        DEVNET_OZ_CLASS_HASH_CAIRO_0,
+        "0xa5d90c65b1b1339",
+        private_key,
+    )
+    .await;
+}
+
+pub async fn deploy_cairo_0_account() {
+    let accounts_file = "tests/data/accounts/accounts.json";
+
+    let contents = fs::read_to_string(accounts_file).expect("Failed to read accounts file");
+    let items: Value = serde_json::from_str(&contents)
+        .unwrap_or_else(|_| panic!("Failed to parse accounts file at = {accounts_file}"));
+
+    let account_data = items
+        .get("alpha-goerli")
+        .and_then(|accounts| accounts.get("cairo0"))
+        .unwrap_or_else(|| panic!("Failed to get cairo0 account"));
+
+    let address = get_from_json_as_str(account_data, "address");
+    let salt = get_from_json_as_str(account_data, "salt");
+    let private_key = get_from_json_as_str(account_data, "private_key");
+
+    let private_key = SigningKey::from_secret_scalar(
+        parse_number(private_key).expect("Failed to convert private key to FieldElement"),
+    );
+
+    deploy_account_helper(address, DEVNET_OZ_CLASS_HASH_CAIRO_0, salt, private_key).await;
+}
+
+async fn deploy_account_helper(
```

Comment:
```suggestion
async fn deploy_test_account(
```
nit: the function name implies we will deploy something that is an account helper, not an account